src/cadence/scripts/getNFTIDs.cdc

```diff
@@ -167,13 +167,13 @@ pub fun main(ownerAddress: Address): {String: [UInt64]} {
     }
 
     if let col = owner.getCapability(KOTD.CollectionPublicPath)
```

Comment:
Can we delete lines 169-177, just realized these are already accounted for on Lines 60 and 84